### PR TITLE
Changes for grants page

### PR DESCRIPTION
**Pull request by RitwikSrivastava**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fixes #<gh-issue-id>

Test URLs:
- Original: https://www.sunstar.com/research/grants-award
- Before: https://main--sunstar--hlxsites.hlx.live/_drafts/ritsriva/grants-award
- After: https://grantspage--sunstar--hlxsites.hlx.page/_drafts/ritsriva/grants-award

- [ ]  **If you are adding a new block or a variation to an existing block**, please fill below:

  Block library path: https://<branch>--sunstar--hlxsites.hlx.page/tools/sidekick/library.html?plugin=blocks&path=/sidekick/blocks/<your-block>&index=0

### CODE REVIEW

File: blocks/cards/cards.css [hunk 1]
```diff
@@ -12,6 +12,24 @@
   text-align: center;
 }
 
+.section .section-container h3 {
```
Comment (sdmcraft) [inline blocks/cards/cards.css:15]: I think these styles will get applied to all `.section .section-container` . Probably you'd want `cards-container`

File: blocks/cards/cards.css [hunk 2]
```diff
@@ -12,6 +12,24 @@
...
+  color: var(--light-black);
+  font-size: var(--heading-font-size-m);
+  font-weight: var(--font-weight-bold);
+  line-height: 2rem;
+  width: 100%;
+  padding-bottom: 1rem;
+}
+
+.section .section-container h1 {
```
Comment (sdmcraft) [inline blocks/cards/cards.css:24]: Same as previous comment.

**Review by sdmcraft — COMMENTED** [review]
Given this is modifying cards styles which is used at other place, you may also want to review the visual differences just to be sure it isn't breaking anything.

**Comment by RitwikSrivastava** [comment]
@sdmcraft Please re-review. I have made the desired changes. Achieved the same page with minimal changes.

**Review by sdmcraft — APPROVED** [review]

**Pull request merged**